src/pages/import-cards-popup/import-cards-popup.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react';
+import { noop } from '../../helpers/func';
+import userEvent from '@testing-library/user-event';
+import { ImportCardsPopup } from './import-cards-popup';
+import { Card } from '../../models/card';
+
+describe('ImportCardsPopup', () => {
+  let mockOnClose: jest.Mock<void, [Card[]]>;
```

Comment:
are we going to add more tests using this?
if not, it makes more sense to live in the test that's using it

Reply:
I was thinking in case we were to add more tests in the future. But I moved it into the test; we can always move it back out again if we end up adding some more, haha